test/ApcTokenStorageTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace {  
```

Comment:
Yes, I understand you're mocking the functions within the namespace. My comment was about declaring
    $mockApcStore = false;
    $mockApcFetch = false;
in the global namespace.  The vars can be declared in the F3\AppNexusClient namespace and will be still visible globally. I mean you don't have to have two namespaces in the file.